packages/css/src/paragraph/paragraph.scss

```diff
@@ -5,28 +5,45 @@
 
 @import "../../utils/calculate-fluid-style";
 
+@import "../../node_modules/@amsterdam/design-system-tokens/dist/variables";
+
 @mixin reset {
   box-sizing: border-box;
   -webkit-text-size-adjust: 100%;
 }
 
 .amsterdam-paragraph {
-  color: var(--amsterdam-color-primary-black);
-  font-family: var(--amsterdam-typography-font-family);
-  font-size: calculate-fluid-style(18, 24);
-  font-weight: var(--amsterdam-typography-font-weight-normal);
-  line-height: calculate-fluid-style(29, 38);
+  color: var(--amsterdam-paragraph-color);
+  font-family: var(--amsterdam-paragraph-font-family);
+
+  /*
+    We don't use the amsterdam-paragraph-font-size variable, because we generate a somewhat
+    complex css rule with a Sass function. The variable is added here so people can
+    overwrite that css rule with their own value.
+  */
+  font-size: var(
+    --amsterdam-paragraph-font-size,
+    calculate-fluid-style($amsterdam-typography-font-size-18, $amsterdam-typography-font-size-24)
+  );
+  font-weight: var(--amsterdam-paragraph-font-weight);
+  line-height: var(--amsterdam-paragraph-line-height);
   margin-block: 0;
+  max-width: 80ch;
```

Comment:
Interesting, but will it scale well? It could for example create unexpected empty space in a grid area. Should we create a prototype to see the effect and discuss with the designers?

Reply:
In case we keep it, let’s encode it into a token.

Reply:
Hm, I see your point. However, if it creates unexpected empty space in a grid area, there's something wrong with the design of the grid area :) Text shouldn't be more than 80 characters wide, this acts as a sort of fail safe for that. 

We can test it on onderzoek.amsterdam.nl and discuss it with the designers.